tests/modules/suite2p_registration/test_suite2p_registration_suite2p_utils.py

```diff
@@ -0,0 +1,136 @@
+import h5py
+import json
+import numpy as np
+from pathlib import Path
+import sys
+import pytest
+import unittest
+from unittest.mock import MagicMock, Mock, patch
+
+has_suite2p = True
+try:
+    import suite2p.registration  # noqa: F401
+except ImportError:
+    # only mock Suite2P if necessary; otherwise, the mock
+    # makes it into the tests that actually rely on Suite2P
+    has_suite2p = False
+    sys.modules['suite2p.registration.rigid'] = Mock()
+    sys.modules['suite2p.registration.register'] = Mock()
+
+
+from ophys_etl.modules.suite2p_registration.suite2p_utils import (  # noqa: E402, E501
+    compute_reference, load_initial_frames)
+
+
+class TestRegistrationSuite2pUtils(unittest.TestCase):
+
+    def setUp(self):
+        file_loc = Path(__file__)
+        self.resource_loc = file_loc.parent / "resources"
+        self.h5_file_loc = self.resource_loc \
+            / "792757260_test_data.h5"
+        self.mock_data_loc = self.resource_loc \
+            / "test_rand.npy"
+        with h5py.File(self.h5_file_loc, 'r') as h5_file:
+            self.frames = h5_file["input_frames"][:]
+            self.ops = json.loads(h5_file['ops'][()].decode('utf-8'))
+            self.original_reference = h5_file["reference_img"][:]
+        self.n_frames = self.frames.shape[0]
+        self.rng = np.random.default_rng(1234)
+
+    def test_load_initial_frames(self):
+        """Test loading specific frames from h5py."""
+        n_frames_to_load = 20
+        loaded_data = load_initial_frames(self.h5_file_loc,
+                                          "input_frames",
```

Comment:
I noticed that sometimes you use `'` and sometimes `"`. I'm not sure which is used in this codebase but try to use whatever is used in this codebase

Reply:
Looks like the code around me uses `'` so I'll switch them to that. Thanks for the catch.